### PR TITLE
added unit tests for michel-codec coverage

### LINKED ISSUE #1969: add unit test coverage for taquito-michel-codec package
the goal is to increase the coverage to the green where possible and to eliminate mutants where practical

### PR DESCRIPTION
**Pull request by michaelkernaghan**
close #1969 

add coverage to get the coverage chart to show less red on it.

### CODE REVIEW

**Comment by michaelkernaghan** [comment]
needs refactoring

**Comment by michaelkernaghan** [comment]
Refactoring is completed.

File: packages/taquito-michel-codec/test/contracts_014.spec.ts [hunk 1]
```diff
@@ -31,15 +37,68 @@ describe('PtKathmandu', () => {
...
-            }
-            throw err;
-          }
+        it('parse check null case', () => {
+          const options: ContractOptions = {
+            protocol: protocol,
+          };
+          const src = '';
+          expect(() => Contract.parse(src, options)).toThrow(InvalidContractError);
+          expect.objectContaining({
```
Comment (roxaneletourneau) [inline packages/taquito-michel-codec/test/contracts_014.spec.ts:48]: missing something here

Comment (michaelkernaghan) [reply packages/taquito-michel-codec/test/contracts_014.spec.ts]: i fixed all of these tests were the throw() was left out

**Review by roxaneletourneau — APPROVED** [review]

**Pull request merged**